champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/sword/Inferno.java

```diff
@@ -85,14 +86,19 @@ public String[] getDescription(int level) {
                 "Release to shoot a scorching blast of fire",
                 "that ignites anything it hits for <stat>" + getFireDuration(level) + "</stat> seconds",
                 "",
-                "Cooldown: <val>" + getCooldown(level)
+                "Cooldown: <val>" + getCooldown(level),
+                "Energy: <val>" + getEnergyPerSecond(level)
         };
     }
 
     public double getFireDuration(int level) {
         return baseFireDuration + level * fireDurationIncreasePerLevel;
     }
 
+    private float getEnergyPerSecond(int level) {
+        return (float) (energy - ((level - 1) * energyDecreasePerLevel));
+    }
+
     public int getNumFlames(int level){
         return baseNumFlames + level * numFlamesIncreasePerLevel;
```

Comment:
fix the scaling here to be level -1, same with getFireDuration above (i know you didnt change those in this PR but might as well fix